src/systemcmds/i2cdetect/Kconfig

```diff
@@ -2,4 +2,11 @@ menuconfig SYSTEMCMDS_I2CDETECT
 	bool "i2cdetect"
 	default n
 	---help---
-		Enable support for i2cdetect
\ No newline at end of file
+		Enable support for i2cdetect
+
+menuconfig USER_I2CDETECT
```

Comment:
This one probably isn't valid for user space?

Reply:
true